.pre-commit-config.yaml

```diff
@@ -0,0 +1,39 @@
+repos:
+    - repo: https://github.com/pre-commit/pre-commit-hooks
+      rev: v4.0.1
+      hooks:
+          - id: check-merge-conflict
+          - id: check-added-large-files
+          - id: check-ast
+          - id: check-symlinks
+          - id: trailing-whitespace
+          - id: check-json
+          - id: debug-statements
+          - id: pretty-format-json
+            args: ["--autofix", "--allow-missing-credentials"]
```

Comment:
Thanks for this - I need to read up a bit more on pre-commit to work out which we need. We might be able to drop the JSON ones.

Reply:
Okay, i will drop the JSON one